CONTRIBUTION.md

```diff
@@ -64,3 +64,16 @@ Bug: T42
 The first line is the subject and should be no longer than 50 characters, the second line is always blank, and other
 lines should be wrapped at 80 characters. This allows the message to be easier to read on GitHub as well as in various
 git tools.
+
+### PR workflow
+
+- Before submitting a PR
+  - Format your code and fix all the spelling/grammatical mistakes.
+  - Limit the column length to 120 characters.
+  - Error messages should not be more than 2 lines long.
+- Following are the things you should keep in mind once you open a PR:
+  - Wait for comments from at least 2 reviewers before pushing changes. Open comments in the meanwhile can be worked on locally.
```

Comment:
Not sure if this within 120 col. I too missed ensuring that in line 33. You are welcome to fix that here if you would like.

Reply:
Will do!